src/js/modules/ContactEmail.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 //TODO: import correct svg
-import AirplaneSVG from 'js/svg/Airplane';
+import EnvelopSVG from 'js/svg/EnvelopO';
```

Comment:
`EnvelopO`? typo?

Reply:
it was envelop-O -- vs filled in... so here we are